heat/core/tensor.py

```diff
@@ -1484,6 +1589,25 @@ def __setitem__(self, key, value):
             raise NotImplementedError(
                 'Not implemented for {}'.format(value.__class__.__name__))
 
+    """
+    This ensures that commutative arithmetic operations work no matter on which side the heat-tensor is placed.
+    
+    Examples
+    --------
+    >>> import heat as ht
+    >>> T = ht.float32([[1., 2.], [3., 4.,]])
+    >>> T + 1
+    tensor([[2., 3.],
+            [4., 5.]])
+    >>> 1 + T
+    tensor([[2., 3.],
+        [4., 5.]])
+    """
+    __radd__ = __add__
+    __rmul__ = __mul__
+
+    # __rfloordiv__  //  TODO: Implement me when implementing __floordiv__
```

Comment:
Please provide both implementations (regular and right-hand side floor-div). Straight forward by adapting div (torch supports truediv as well)